meta-ivi/recipes-extended/persistence-common-object/persistence-common-object_1.0.3.bb

```diff
@@ -20,4 +20,8 @@ inherit autotools-brokensep pkgconfig
 
 EXTRA_OECONF = " --with-database=key-value-store "
 
-FILES_${PN} += "${datadir}"
+do_install_append() {
+    rm -rf ${D}${sysconfdir}
```

Comment:
Is this safe or could it interact with other packages?  
And why is it needed to do this instead of not touching it?

Reply:
It should not interact with other packages as it's done on the installation destination dir for that specific package, ie, build/tmp/work/<architecture>/<package>/<version>/$sysconfdir

bitbake fails with an error if there are installed files that are not added to any FILES_x directives.

Reply:
If you have a better solution I'm all ears :)

Reply:
I guess the optimal solution would be if only one of the packages actually installed that file though :)